lib/wovnrb/html_replacers/replacer_base.rb

```diff
@@ -1,7 +1,8 @@
 module Wovnrb
   class ReplacerBase
-    def initialize(store)
+    def initialize(store, ignored_class_set = nil)
       @store = store
+      @ignored_class_set = Set.new(ignored_class_set || [])
```

Comment:
could `ignored_class_set` be by default an empty array instead of this check?

Reply:
`ignored_class_set`'s default is nil, and `Set.new(nil)` is empty Set, but I still prefer to add `Set.new(ignored_class_set || [])`

Reply:
but if you set the `ignored_class_set = []` as the default you don't have to do the `||`. Makes the code a little easier to read less logic.

Reply:
I changed to `ignored_class_set = []` here https://github.com/WOVNio/wovnrb/pull/128/commits/1eac217bee5ca85cf3fb5307127093a057cae5ee 👍